src/services/registrationData.ts

```diff
@@ -103,22 +103,173 @@ export async function upsertRegistrationData({
 }
 
 /**
- * Function: updateQuestionOptions
- * Updates or inserts registration data into the question options table.
- * @param dbPool - The database pool instance of type `PostgresJsDatabase<typeof db>`.
- * @param registrationData - An array of objects representing registration data.
- *   Each object should have the properties:
- *   - id: The unique identifier for the registration data.
- *   - registrationFieldId: The identifier for the registration field associated with the data.
- *   - value: The value of the registration data.
- * @returns A Promise that resolves once the update/insert operation is completed.
+ * Fetches registration fields from the database.
+ * @param dbPool - The database pool instance.
+ * @param registrationFieldIds - Array of registration field IDs.
+ * @returns An array of registration fields.
  */
-export async function updateQuestionOptions(
+async function fetchRegistrationFields(
+  dbPool: PostgresJsDatabase<typeof db>,
+  registrationFieldIds: string[],
+): Promise<
+  {
+    registrationFieldId: string;
+    questionId: string;
+    questionOptionType: string;
+  }[]
+> {
+  // Fetch registration fields from the database based on the provided IDs
+  const registrationFields = await dbPool.execute<{
+    registrationFieldId: string;
+    questionId: string;
+    questionOptionType: string;
+  }>(
+    sql.raw(`
+        SELECT id AS "registrationFieldId", question_id AS "questionId", question_option_type AS "questionOptionType"
+        FROM registration_fields
+        WHERE question_id IS NOT NULL
+        AND question_option_type IN ('TITLE', 'SUBTITLE')
+        AND id IN (${registrationFieldIds.map((id) => `'${id}'`).join(', ')})
+        `),
```

Comment:
we can migrate this to drizzle selects as it is not that complex and can help us remove a possible bug in the future

Reply:
Done by @diegoalzate :)